components/AppHeader.vue

```diff
@@ -1,18 +1,5 @@
 <template>
   <header>
```

Comment:
Should we hide the banner with a `display:none` but keep the markup or move the banner to a separate component? We'll probably need to include banners in the future.

Reply:
Instead of display none, I created a separate component so we can add it and remove it from markup whenever we want
Commit: https://github.com/arcana-network/website/pull/98/commits/d4340036404f1d237a0e3debb96ad72114d84ae2